product_pack/models/product_pack_line.py

```diff
@@ -56,4 +56,4 @@ def _check_recursion(self):
 
     def get_price(self):
         self.ensure_one()
-        return self.product_id.price * self.quantity
+        return self.product_id.lst_price * self.quantity
```

Comment:
lst_price in not exactly the same as price in v15, probably you should call `_get_contextual_price`() instead. Related Odoo PR: https://github.com/odoo/odoo/pull/79605